CASMREL-847/lib/version.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Hewlett Packard Enterprise Development LP
+
+: "${RELEASE:="${RELEASE_NAME:="casmrel-847"}-${RELEASE_VERSION:="0.9.29999"}"}"
```

Comment:
version: TBD

Reply:
I'd say the `RELEASE_VERSION` of this patch would be 1.0.0 when released.